src/rpc.rs

```diff
@@ -207,9 +222,78 @@ pub mod kate {
 		Ok(value)
 	}
 
-	pub async fn query_rows(client: &Client, rows: Vec<u32>, at: Option<H256>) -> Result<Vec<GRow>, subxt::Error> {
+	pub async fn query_multi_proof(
+		client: &Client,
+		at: Option<H256>,
+		cells: Vec<Cell>,
+	) -> Result<(Vec<(GMultiProof, GCellBlock)>, Vec<u8>), subxt::Error> {
+		let header = chain::get_header(client, at).await?;
+
+		let Some((_, _, _, commitment)) = extract_kate(&header.extension) else {
+			return Err(subxt::Error::Other(
+				"Skipping block without header extension".to_string(),
+			));
+		};
+
+		let params = rpc_params![cells.to_vec(), at];
+		let proofs: Vec<(GMultiProof, GCellBlock)> = client.rpc_client.request("kate_queryMultiProof", params).await?;
+
+		Ok((proofs, commitment))
+	}
+
+	pub async fn generate_pmp() -> M1NoPrecomp<Bls12_381, BlstMSMEngine> {
+		multiproof_params()
+	}
+
+	pub async fn verify_multi_proof(
```

Comment:
Why do we need this in the sdk? Isn't this something that should be in avail-core?